src/NHSD.BuyingCatalogue.Identity.Api/Controllers/AccountController.cs

```diff
@@ -35,24 +35,21 @@ public sealed class AccountController : Controller
             DisabledErrorMessageSettings disabledErrorMessageSettings,
             PublicBrowseSettings publicBrowseSettings)
         {
-            _loginService = loginService;
-            _logoutService = logoutService;
-            _passwordResetCallback = passwordResetCallback;
-            _passwordService = passwordService;
-            _disabledErrorMessageSettings = disabledErrorMessageSettings;
-            _publicBrowseSettings = publicBrowseSettings;
+            this.loginService = loginService;
+            this.logoutService = logoutService;
+            this.passwordResetCallback = passwordResetCallback;
+            this.passwordService = passwordService;
+            this.disabledErrorMessageSettings = disabledErrorMessageSettings;
+            this.publicBrowseSettings = publicBrowseSettings;
         }
 
         [HttpGet]
         public IActionResult Login(Uri returnUrl)
         {
-            if (returnUrl == null)
+            if (returnUrl is null)
                 return RedirectToPublicBrowseLogin();
 
-            LoginViewModel loginViewModel = new LoginViewModel
-            {
-                ReturnUrl = returnUrl
-            };
+            LoginViewModel loginViewModel = new LoginViewModel { ReturnUrl = returnUrl };
```

Comment:
can this be updated to `new()`?

Reply:
Yes, it can although I didn't get any prompts for it (either from the built-in VS analyser or ReSharper) for some reason, which is why I missed it. However, rather than use `new()` I'll use `var` instead.

Reply:
Good spot by the way!